src/Mailgun/Connection/RestClient.php

```diff
@@ -349,6 +349,14 @@ public function setApiVersion($apiVersion)
     }
 
     /**
+     * @return string
+     */
+    public function getApiVersion()
```

Comment:
From my POV I cannot see the problem to implement a getter. Given we already have a setter, a getter is not a bad idea. @Nyholm why do you think this is not a good idea?